lib/modules.nix

```diff
@@ -616,7 +659,7 @@ rec {
   fixupOptionType = loc: opt:
     let
       options = opt.options or
-        (throw "Option `${showOption loc'}' has type optionSet but has no option attribute, in ${showFiles opt.declarations}.");
+        (throw "Option `${showOption loc}' has type optionSet but has no option attribute, in ${showFiles opt.declarations}.");
```

Comment:
Is this the place that was buggy?

Reply:
Exactly. Scope checking will now find undeclared local variables, even if not used in any tests.